kit-sdq/programming-lecture-eclipse-artemis
# Automatic correction support

### Checklist
- [x] I tested *all* changes and their related features locally or with a test instance of Artemis.
- [ ] I documented the changes in the Documentation (/docs).

### Motivation and Context
This PR exists mainly to trigger a build of the plugin with Autograder integration.

Automatic correction may contribute to making the assessment process more fair and reduce the workload of tutors.

### Description
Include the Autograder and execute it automatically on each submission. The implementation is currently extremely dirty and more or less hardcoded for the current plugin configuration. Because of problems with Maven dependencies and Eclipse the Autograder is currently included as a Jar file which has to be manually updated whenever the Autograder itself is updated.

### Steps for Testing
Start a new assessment, wait a few seconds and observe the magic of automatic correction!
